code/modules/bluespace_exploration/exploration_pinpointer.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/pinpointer/exploration
+	name = "exploration pinpointer"
+	desc = "A handheld tracking device that locks onto the signal emitted by a bluespace drive. You can link it with a bluespace drive."
+
+/obj/item/pinpointer/exploration/Initialize()
+	. = ..()
+	if(GLOB.main_bluespace_drive)
```

Comment:
Should probably set this when they turn it on in attack_self, with some error message for the thing being destroyed.